src/plugins/permissionFreeWill/index.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { definePluginSettings } from "@api/Settings";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+
+const settings = definePluginSettings({
+    lockout: {
+        type: OptionType.BOOLEAN,
+        default: true,
+        description: 'Bypass the permission lockout prevention ("Pretty sure you don\'t want to do this")',
+        restartNeeded: true
+    },
+    onboarding: {
+        type: OptionType.BOOLEAN,
+        default: true,
+        description: 'Bypass the onboarding requirements ("Making this change will make your server incompatible [...]")',
+        restartNeeded: true
+    }
+});
+
+export default definePlugin({
+    name: "PermissionFreeWill",
+    description: "Disables the client-side restrictions for channel permission management.",
+    authors: [Devs.lewisakura],
+
+    patches: [
+        // Permission lockout, just set the check to true
+        {
+            find: "Messages.SELF_DENY_PERMISSION_BODY",
+            replacement: [
+                {
+                    match: /"Unexpected boolean action".{5,20}allow.{5,20}.{80,100}if\(/,
```

Comment:
this does not seem very robust.
It's supposed to match this if (C.Z.can), right? 
![grafik](https://github.com/Vendicated/Vencord/assets/45497981/8d3d70ba-55e5-4b7a-90dd-7a712b124d84)

much more robust would be smth like
```suggestion
                    match: /case "DENY":.{0,50}if\(\i\.\i\.can/,
```